dev/PushNotifications/PushNotificationsLongRunningTask/NotificationListener.h

```diff
@@ -1,19 +1,29 @@
 ﻿#pragma once
 
 #include <FrameworkUdk/PushNotificationsRT.h>
+#include <FrameworkUdk/ToastNotificationsRT.h>
+#include "ToastRegistrationManager.h"
 
-class NotificationListener : public Microsoft::WRL::RuntimeClass<::ABI::Microsoft::Internal::PushNotifications::INotificationListener>
+class NotificationListener : public Microsoft::WRL::RuntimeClass<::ABI::Microsoft::Internal::PushNotifications::INotificationListener,
+                                                                    ::ABI::Microsoft::Internal::PushNotifications::INotificationListener2>
```

Comment:
Future: This should be a cppwinrt type; avoid using WRL unless required.

Reply:
https://github.com/microsoft/WindowsAppSDK/issues/2221

Going to look into this in the future.